### PR TITLE
Add official markets ENS record to the Polygon proposal

### PR DESCRIPTION
**Pull request by jflatow**

### CODE REVIEW

File: deployments/polygon/usdc/migrations/1675200105_supply_caps_and_ens.ts [hunk 1]
```diff
@@ -70,11 +74,34 @@ export default migration('1675200105_raise_polygon_supply_caps', {
...
+    const ENSResolver = await deploymentManager.existing('ENSResolver', ENSResolverAddress);
+    const subdomainHash = ethers.utils.namehash(ENSSubdomain);
+    const officialMarketsJSON = await ENSResolver.text(subdomainHash, ENSTextRecordKey);
+    const officialMarkets = JSON.parse(officialMarketsJSON);
+    officialMarkets[137] = [
+      {
+        baseSymbol: 'USDC',
+        cometAddress: comet.address,
+      }
+    ];
```
Comment (kevincheng96) [inline deployments/polygon/usdc/migrations/1675200105_supply_caps_and_ens.ts:87]: 🆒

**Review by kevincheng96 — APPROVED** [review]
Looks good!

File: scenario/context/CometContext.ts [hunk 2]
```diff
@@ -407,6 +407,7 @@ async function forkContext(c: CometContext, w: World): Promise<CometContext> {
 export const constraints = [
   new FilterConstraint(),
   new NativeTokenConstraint(),
+  new ProposalConstraint(), // XXX
```
Comment (jflatow) [inline scenario/context/CometContext.ts:410]: Note: remove this after ENS proposal passes, want to assume that proposal has run in the migration script

**Review by scott-silver — APPROVED** [review]
Looks good!

Does verify never get called anywhere in `tasks/deployment_manager/task.ts`? Is a migration's `verify` function only ever run in scenarios?

**Comment by jflatow** [comment]
> Looks good!
> 
> Does verify never get called anywhere in `tasks/deployment_manager/task.ts`? Is a migration's `verify` function only ever run in scenarios?

Yeah it only runs in scenarios because it has to run after the proposal actually executes

**Comment by jflatow** [comment]
Moved the full ENS setup into this proposal since the original proposal with just the initial setup failed to reach quorum.

**Review by kevincheng96 — APPROVED** [review]

**Pull request merged**